orangecontrib/timeseries/widgets/tests/test_owtabletotimeseries.py

```diff
@@ -13,6 +14,19 @@ class TestAsTimeSeriesWidget(WidgetTest):
     def setUp(self):
         self.widget = self.create_widget(OWTableToTimeseries)  # type: OWTableToTimeseries
 
+    def test_time_as_metas(self):
+        """
+        As Timeseries should accept attributes from X and metas.
+        """
+        w = self.widget
+        data = Table("philadelphia-crime.csv.xz")[:20]
```

Comment:
https://travis-ci.org/biolab/orange3-timeseries/builds/451775988#L708-L720

Reply:
Dude, I was literally testing this afternoon. With that same data set? Where did it go? 😮 And what is a good simple alternative dataset to test with?